### ISSUE TITLE
Test CLI options not being passed

### ISSUE THREAD

**Issue by mrberggg** [vuejs/vue-cli#1109]
### Version
3.0.0-beta.6

### Reproduction link
[https://gist.github.com/mrberggg/d43f2d0373e9c22c5006bb763b2295ad](https://gist.github.com/mrberggg/d43f2d0373e9c22c5006bb763b2295ad)

### Steps to reproduce
In a new project with Jest tests, run `npm run test --watch`. Tests will run without the --watch flag. Run `./node_modules/.bin/vue-cli-service test --watch`. Tests will run with the --watch flag correctly passed.

### What is expected?
CLI options should be passed to Jest

### What is actually happening?
CLI options are not being passed to Jest

**Comment by BenoitAverty** [comment]
that's actually a feature of NPM I think.

Have you tried to run `npm run test -- --watch` ? The `--` instructs npm to pass all following options to the underlying command. Without it, npm considers the `--watch` to be an option of `npm run`.

**Comment by mrberggg** [comment]
@BenoitAverty You're right, that does the trick. Not the best dev experience though and seems like the docs are currently misleading, indicates that passing options should work same as passing them to jest. Seems like watching is common enough to include some workaround, even perhaps adding another package.json script.

For reference, here's the current docs for using jest options:
`Usage: vue-cli-service test [options] <regexForTestFiles>`

**Comment by LinusBorg** [comment]
@mrberggg You are right, the docs could maybe clearer here, probably because Evan, using yarn, didn't have this oddity about npm in mind. We could add a small hint about this.

On the other hand, we would have to add this to each and every plugin that adds a command, as well as the build-in ones, essentially educating people about how npm works in dozens of places throughout the docs. I'm not sure we want that.

And technically, the docs are *correct*: As you explain yourself in the OP, when you run `vue-cli-service` directly, passing --watch works, and that's what the docs show: a direct call of the executable.
